mq/plugins/cloudtrail.py

```diff
@@ -117,6 +118,10 @@ def onMessage(self, message, metadata):
         if not message['source'] == 'cloudtrail':
             return (message, metadata)
 
+        if 'requestparameters' in message['details']:
+            if 'htmlpart' in message['details']['requestparameters']:
+                message['details']['requestparameters']['htmlpart'] = message['details']['requestparameters']['htmlpart'][0:4095]
```

Comment:
Could we define a constant that describes the meaning of the value 4095 here?

Reply:
I'll add a comment to the code to help.